syncapi/storage/shared/syncserver.go

```diff
@@ -1036,17 +538,6 @@ func (d *Database) UpsertRoomUnreadNotificationCounts(ctx context.Context, userI
 	return
 }
 
-func (d *Database) GetUserUnreadNotificationCountsForRooms(ctx context.Context, userID string, rooms map[string]string) (map[string]*eventutil.NotificationData, error) {
```

Comment:
So am I right in saying that all DB functions which are used by the `/sync` side are now in `transaction.go` and all DB functions used by the roomserver consumer remain here?

Reply:
That's pretty much it, yes.